app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def av_actor_age
+    actors.average(:age)
+  end
+
+  def order_actor_name
+    actors.order(:name)
```

Comment:
This user story asked for this to be sorted by age, youngest to oldest, not name. Technically this would mean the user story is not complete (and therefore not a passing midmod), however the rest of your work is very thorough and I don't think that would be a fair representation of your capabilities. I'm going to let it slide since I'm sure it was just a misreading, but keep an eye out for those details.